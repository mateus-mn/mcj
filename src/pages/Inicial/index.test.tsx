import { render } from '@testing-library/react';
import { QueryClient, QueryClientProvider } from 'react-query';
import InicialPage from '.';

describe('Testes da pÃ¡gina Inicial', () => {
	it('Teste de snapshot', () => {
		const queryClient = new QueryClient();
		const { container } = render(
			<QueryClientProvider client={queryClient}>
				<InicialPage />
			</QueryClientProvider>,
		);

		expect(container).toMatchSnapshot();
	});
});
