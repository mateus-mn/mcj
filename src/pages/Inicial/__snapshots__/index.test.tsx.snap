// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testes da p√°gina Inicial Teste de snapshot 1`] = `
<div>
  <div
    class="text-center"
  >
    <div
      class="spinner-border"
      role="status"
    >
      <span
        class="visually-hidden"
      >
         Carregando... 
      </span>
    </div>
  </div>
</div>
`;
